packages/cli/src/commands/ui/create/vue.ts

```diff
@@ -91,11 +91,11 @@ export default class Vue extends Command {
       '--orgId',
       cfg.organization,
       '--apiKey',
-      cfg.accessToken!,
+      apiKey.value!,
       '--platformUrl',
       platformUrl({environment: cfg.environment}),
       '--user',
-      providerUsername,
+      userInfo.providerUsername,
```

Comment:
Why `userInfo.providerUsername` and not `userInfo` (I don't see what changed to justify it) ?

Reply:
Because of the method `platformUserCredentials` it returns a `{UserModel, ApiKeyModel}`

Reply:
Oh I get it!
I missed the deconstruction that was happening before with `getUserInfo`
i.e. I read ` const {providerUsername} = await this.getUserInfo();` as ` const providerUsername = await this.getUserInfo();`

Which confused me when checking your changeset.

My bad!